server/models/Request.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
```

Comment:
Would be better to follow conventions like camelCase for JavaScript and also userId should given as reference to USER schema

Reply:
I have taken note of the camelCase suggestions and made the necessary adjustments, thanks

Reply:
With regards to embedding the USER document in the REQUEST document, I did think of that but I was concerned because USER was exported like this `module.exports = USER`, and to make use of the user schema in the Request model, I would need to export it like this `module.exports = {USER, userSchema}` this would mean I'd have to check all files where the USER is imported and modify them since USER is no longer a default export. I agree that embedding would be the best way to go about it though, And I'll check to see if a way exists to update all references to a variable using vscode.

Reply:
Hi @BerryGIT-ME, Ajay was referring to adding a reference to the `User` model in the userId/sitterId fields.

Example: `[{ type: Schema.Types.ObjectId, ref: 'Person' }]`
Ref Link: https://mongoosejs.com/docs/populate.html